frb_codegen/src/ir/ty_delegate.rs

```diff
@@ -30,6 +30,7 @@ pub enum IrTypeDelegate {
     Array(IrTypeDelegateArray),
     String,
     StringList,
+    DartOpaque(IrTypeDartOpaque),
```

Comment:
Btw curious why are we impl it as delegate?

Reply:
Well of course it is possible not to delegate. fixed